src/modules/assessments/components/AssessmentForm.tsx

```diff
@@ -338,7 +338,12 @@ const AssessmentForm: React.FC<Props> = ({
   return (
     <Grid container spacing={2} sx={{ pb: 20, mt: 0 }}>
       {showNavigation && navigation}
-      <Grid item xs={showNavigation ? 9.5 : 12} sx={{ pt: '0 !important' }}>
+      <Grid
+        id='focusable-form'
```

Comment:
maybe we can pass this as a prop instead so its not a hard-coded string? it's a little confusing to search the codebase and find four usages of `focusable-form`, but I _think_ they are actually 2 separate pairs that have nothing to do with each other.

could this be implemented as:

```
const formContainerId = 'assessment-form-container'

..
<AssessmentFormSideBar formFocusTargetId={formContainerId} />
...
<Grid id={formContainerId}>

Reply:
Yes agreed, thanks! In https://github.com/greenriver/hmis-frontend/pull/1018/commits/d127879d3dda7a86ce59a03eb3ec6d5fcf7d4374 I factored this into a shared variable to avoid repeating the hardcoded string. Otherwise I think your comment is partly addressed with the refactor, since there are no longer 2 separate pairs

Reply:
This same feedback applies to `focusable-main`, which has 2 instantiations, 1 in dashboard context and 1 in non-dashboard context (such as all clients). I've made a maybe-satisfactory attempt here https://github.com/greenriver/hmis-frontend/pull/1018/commits/c39393ad6ac69c17cd8e89da27cc7ade761f8b4e, let me know what you think